src/spaces/utilities.jl

```diff
@@ -163,12 +163,12 @@ spaces this clamps the position to the space extent.
 normalize_position(pos, model::ABM) = normalize_position(pos, model.space)
 
 function normalize_position(pos, space::ContinuousSpace{D,true}) where {D}
-    return mod.(pos, spacesize(space))
+    return mod.(pos, SVector(spacesize(space)))
 end
 
 function normalize_position(pos, space::ContinuousSpace{D,false}) where {D}
```

Comment:
This functions could be written as:
```
function normalize_position(pos::T, space::ContinuousSpace{D,false}) where {T,D}
    ss = spacesize(space)
    return T(clamp.(pos, 0.0, prevfloat.(ss)))
end
```

to support Tuples, SVectors, and possibily FieldVectors as well.